module1/homework/grayscaleImages/test.cpp

```diff
@@ -1,11 +1,10 @@
 #include <algorithm>
 #include <array>
+#include "compression.hpp"
+#include "gtest/gtest.h"
 #include <utility>  // for std::pair<>
 #include <vector>
```

Comment:
Why have you put headers in between includes from standard library. They should be in another paragraph. Check Q&A#1 lesson,where there is section for includes order.